core/src/layers/mod.rs

```diff
@@ -119,6 +119,11 @@ mod dtrace;
 #[cfg(all(target_os = "linux", feature = "layers-dtrace"))]
 pub use self::dtrace::DtraceLayer;
 
+#[cfg(feature = "layers-foyer")]
+mod foyer;
+#[cfg(feature = "layers-foyer")]
+pub use self::foyer::FoyerLayer;
```

Comment:
I think we need to expose a `FoyerValue` here so users can construct an instance.